platform/feature-api/communication/remote-pod/src/remote.ts

```diff
@@ -257,25 +255,14 @@ export class RemoteClientPod implements Pod {
 
     get endpoint(): Endpoint {
         return {
-            send: (
-                endpointId: string,
-                featureIdToken: string,
-                payload: string,
-                contentType?: string,
-                authorization?: string
-            ) =>
-                this.rpcClient
-                    .endpoint()
-                    .send(endpointId, featureIdToken, payload, contentType, authorization)(),
-            get: (
-                endpointId: string,
-                featureIdToken: string,
-                contentType?: string,
-                authorization?: string
-            ) =>
-                this.rpcClient
-                    .endpoint()
-                    .get(endpointId, featureIdToken, contentType, authorization)(),
```

Comment:
@RichardSto @JJ Thanks for your feedback! 
glad to see we've a consensus hehe... just to confirm about the changes you request:
we're dropping the `featureIdToken` and leave the args as they're. Just to highlight here than on calls like these ones ^, this drop brings a different sequence of args... are we fine for this? won't break anything? 
Same in `browserPod` calls, btw